src/main/java/org/sonarlint/intellij/config/global/SonarLintGlobalSettings.java

```diff
@@ -87,6 +88,22 @@ public boolean isSecretsNeverBeenAnalysed() {
     return secretsNeverBeenAnalysed;
   }
 
+  public void setLastAutoAnalysisDate(long lastAutoAnalysisDate) {
+    this.lastAutoAnalysisDate = lastAutoAnalysisDate;
+  }
+
+  public void setReportUsed(boolean reportUsed) {
+    isReportUsed = reportUsed;
+  }
+
+  public long getLastAutoAnalysisDate() {
+    return lastAutoAnalysisDate;
+  }
+
+  public boolean isReportUsed() {
+    return isReportUsed;
+  }
+
```

Comment:
Is there a way to not store those details in the global settings? I think we should stick to user preferences here